website/docs/utilities/fixupx.md

```diff
@@ -0,0 +1,37 @@
+---
+title: FixupX Reminder
+---
+
+Replies with fixupx.com links to messages containing x.com links.
+
+## Trigger
+
+**Type:** `Regex`<br />
+**Trigger:** `https:\/\/x\.com(?:\/[^\s]*)?`<br />
+
+## Usage
+
+Post any message with a link to [x.com](https://x.com/).
+
+### Example
+
+```txt
+Check out this Tweet! https://x.com/rickastley/status/890617797956456448
+```
+
+## Configuration
+
+In the custom command control panel, configure the command as follows:
+
+**Trigger type:** `Contains`  
+**Trigger:** `https://x.com`
+
```

Comment:
Doesn't this contradict the above regex trigger instructions?

Reply:
I was under the impression that the Trigger section is for the code itself, and not necessarily for the configuration in the control panel. But I see my mistake now - will get that corrected!